internal/handlers/common/params.go

```diff
@@ -186,7 +204,14 @@ func GetSkipStageParam(value any) (int64, error) {
 
 // getBinaryMaskParam matches value type, returning bit mask and error if match failed.
 // Possible values are: position array ([1,3,5] == 010101), whole number value and types.Binary value.
-func getBinaryMaskParam(mask any) (uint64, error) {
+//
+// Returned errors:
+//   - ErrBadValue if positional argument is not a number;
```

Comment:
```suggestion
//   - ErrBadValue if positional argument is not int32;
```

Reply:
Fixed. It could be `[int, long, double]`.